src/System.Management.Automation/utils/PowerShellETWTracer.cs

```diff
@@ -1085,7 +1085,7 @@ public bool TraceJob(Job job)
             {
                 return DebugChannel.TraceDebug(PowerShellTraceEvent.Job,
                                                PowerShellTraceOperationCode.Method, PowerShellTraceTask.None,
-                                               job.Id.ToString(CultureInfo.InvariantCulture), job.InstanceId.ToString(), "NULL job");
+                                               "", "", "NULL job");
```

Comment:
I have a slight preference for `""` over `String.Empty`.  A constant should generate less IL, though the net result might not matter as the JIT has special knowledge of `String.Empty`.